app/templates/components/dispatch-form.hbs

```diff
@@ -96,7 +97,7 @@
         <div class="left-icon"><i>{{fa-icon 'exclamation-triangle'}}</i></div>
         <div class="right-message">
           <div>{{t "dispatch.separate_set"}}</div>
-          <div>You could dispatch the {{#link-to 'items.detail' item}}entire set{{/link-to}} instead.</div>
+          <div> You could dispatch the {{#link-to 'items.detail' item}}entire set{{/link-to}} instead.</div>
```

Comment:
missing translation here

Reply:
yeah, @steveyken i was not sure whether this link-to will work or not (if added in translations). So I purposely left it. 
Will try to check how we can work this out. 
Thanks